saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/infoFormSection.js

```diff
@@ -191,7 +194,16 @@ function InfoFormSection(options) {
                  
                  if ($optionSelected.index() > 0) {
                      
-                     html += addPhrase($optionSelected.text(), fieldDescription, comma);
+                     var optionSelectedText = $optionSelected.text();
+                     
+                     if ($this.attr("id") === "select-common-court") {
+                         if ($("#ccIncludeDependentCourtJudgments").is(":checked")) {
+                             optionSelectedText = optionSelectedText + $("#ccIncludeDependentCourtJudgments").attr("data-info-section-custom-text");
```

Comment:
Here you need to also add space (**+ " "**):
`optionSelectedText = optionSelectedText + " " + $("#ccIncludeDependentCourtJudgments").attr("data-info-section-custom-text");`